src/main/kotlin/no/nav/syfo/dialogmote/DialogmotedeltakerVarselJournalpostService.kt

```diff
@@ -31,6 +31,56 @@ class DialogmotedeltakerVarselJournalpostService(
         }
     }
 
+    fun getDialogmotedeltakerArbeidsgiverVarselForJournalforingList(): List<Triple<Virksomhetsnummer, PersonIdentNumber, DialogmotedeltakerArbeidsgiverVarsel>> {
+        return getDialogmotedeltakerArbeidsgiverVarselWithoutJournalpostList().filter { (_, _, arbeidsgiverVarsel) ->
+            journalforingVarselTypeList.contains(arbeidsgiverVarsel.varselType)
```

Comment:
Kunne vi spart oss noen kall til databasen ved å gjøre denne filtreringen før vi henter motedeltakere i metoden under?

Reply:
Nei, ikke slik det er nå. I praksis er denne filtreringen redundant siden vi journalfører alle varseltyper (bortsett fra referat som journalføres for seg, men de vil heller ikke være med i listen ovenfor *før* filtreringen).